docs/setup.rst

```diff
@@ -468,8 +468,36 @@ writing. Windows is not and will never be supported.
     to compile this by yourself, because this software has been patented until around 2017 and
     binary packages are not available for most distributions.
 
-Migration to paperless-ngx
-#########################
+Migrating to Paperless-ngx
+##########################
+
+Migration is possible from both Paperless-ng or directly from the 'original' Paperless.
+
+Migrating from Paperless-ng
+===========================
+
+Paperless-ngx is meant to be a drop-in replacement for Paperless-ng and thus upgrading should be
+trivial for most users, especially when using docker. However, as with any major change, it is 
+recommended to take a full backup first. Once you are ready, simply change the docker image to
+point to the new source, e.g. change:
+
+.. code::
+
+  image: jonaswinkler/paperless-ng:latest
+
+to
+
+.. code::
+
+  image: ghcr.io/paperless-ngx/paperless-ngx:latest
+    
+Then restart your container and that's it! 
```

Comment:
We should probably be more explicit (`docker-compose pull && docker-compose up -d`) for users who are not that familiar with docker.

Reply:
Actually I disagree. First of all, they might not be using compose and also this ain’t a docker guide 😁

Reply:
But you’re right it should be more explicit that they have to pull the new container

Reply:
Hm, the line `image: ...` only makes sense with compose, no? Also our installation guide explicitly asks users to use compose and judging from some comments on matrix not everyone will know how to "restart the container". Thinking about it, it may not even be the correct term. `docker-compose restart` will not use the new image, right 🤔

Reply:
How about we instead add a reference to our update guide: https://paperless-ngx.readthedocs.io/en/latest/administration.html#docker-route